### DISCUSSION TITLE
Welcome to quick-symlink Discussions!

### DISCUSSION THREAD

**Discussion by ololx** [ololx/quick-symlink#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shanti099** [comment]
Can we expect a Windows version, please?

**Reply by ololx** [reply]
Hello,
Thank you for your interest.
I'm afraid, at the moment I can't answer that question. However, I will try to explore the possibilities of porting quick-symlink to Windows and get back to you later.

**Reply by Shanti099** [reply]
Thank you so much. 
Very useful tool and and handily made through the context menu.
There are other tools, but they do not work as realized in your development.
